### PR TITLE
Adding ruff to ci.yml

### PR DESCRIPTION
**Pull request by alckasoc**

### CODE REVIEW

**Comment by talmo** [comment]
Are there supposed to be any outputs from ruff after linting @alckasoc? (Looking at the Action run logs.)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -55,9 +55,13 @@ jobs:
       run: |
         black --check sleap_nn tests
 
-    - name: Run pydocstyle
+    # - name: Run pydocstyle
+    #   run: |
+    #     pydocstyle --convention=google sleap_nn/
```
Comment (talmo) [inline .github/workflows/ci.yml:60]: Remove unused lines

File: pyproject.toml [hunk 2]
```diff
@@ -56,6 +57,16 @@ Repository = "https://github.com/talmolab/sleap-nn"
 [tool.black]
 line-length = 88
 
-[pydocstyle]
+# [pydocstyle]
```
Comment (talmo) [inline pyproject.toml:60]: Remove unused lines

**Review by talmo — CHANGES_REQUESTED** [review]

**Comment by alckasoc** [comment]
I don't think ruff has a "No errors found." output like Black. There's a `--quiet` flag which suppresses all ruff outputs except for errors.

**Review by talmo — APPROVED** [review]

**Pull request merged**